src/api.ts

```diff
@@ -141,13 +147,28 @@ export interface FernsRouterOptions<T> {
   /** Hook that runs after the object is fetched but before it is serialized.
    * Returns a promise so that asynchronous actions can be included in the function.
    * Throw an APIError to return a 400 with an error message.
+   * @deprecated: Use responseHandler instead.
    */
   postGet?: (value: T, request: express.Request) => void | Promise<T>;
   /** Hook that runs after the list of objects is fetched but before they are serialized.
    * Returns a promise so that asynchronous actions can be included in the function.
    * Throw an APIError to return a 400 with an error message.
+   * @deprecated: Use responseHandler instead.
+   */
+  postList?: (
+    value: (Document<any, any, any> & T)[],
+    request: express.Request
+  ) => Promise<(Document<any, any, any> & T)[]>;
+  /** Serialize an object or list of objects before returning to the client. This is a good spot to remove sensitive
+   * information from the object, such as passwords or API keys.
+   * Throw an APIError to return a 400 with an error message.
    */
-  postList?: (value: Document<T, {}, {}>[], request: express.Request) => Promise<Document[]>;
+  responseHandler?: (
```

Comment:
Just the postList/postGet handlers, because those are kinda weird. It also deprecates/replaces any existing serializers (which I don't think we're really using). You *could* put all your postCreate/postUpdate calls in the responseHandler but it'd be a bit weird, and delete doesn't go through responseHandler (there's no response), so that can't be replaced.